src/thunderbots/software/ai/passing/pass_generator.h

```diff
@@ -72,6 +79,16 @@ namespace AI::Passing
          */
         void setPasserPoint(Point passer_point);
 
+        /**
+         * Set the id of the robot performing the pass.
```

Comment:
Just to be explicit, indicate we assume this robot is on the friendly team

Reply:
specified.